### PR TITLE
feat(server): switch to webpack-dev-server

### PR DESCRIPTION
**Pull request by cbourgois**
* Switch to `webpack-dev-server`.
* Change port to `9000`
* Add proxy to local 2API by using `--env.local2API`.

Start command become `yarn start --env.local2API`

### CODE REVIEW

File: packages/ovh-manager/server/proxy/Aapi/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const config = {
+  aapi: {
+    url: 'http://localhost:8080',
+  },
+};
+
+module.exports = {
+  context: ['/engine/2api', '/2api'],
```
Comment (jleveugle) [inline packages/ovh-manager/server/proxy/Aapi/index.js]: why declare 2 paths `/engine/2api` and `2api`, we should only use `/engine/2api` (as production, no ?

File: packages/ovh-manager/server/proxy/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  Aapi: require('./Aapi'), // eslint-disable-line global-require
```
Comment (jleveugle) [inline packages/ovh-manager/server/proxy/index.js]: noooooooooooooo ! :)

You should do the `require` at the top of file, and use the created var instead :)

File: packages/ovh-manager/webpack.config.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
 const merge = require('webpack-merge');
 const common = require('./webpack.common.js');
 
-module.exports = merge(common, process.env.WEBPACK_SERVE ? require('./webpack.dev.js') : require('./webpack.prod.js'));
+module.exports = (opts = {}, env = {}) => merge(common, env.production ? require('./webpack.prod.js') : require('./webpack.dev.js')(opts)); // eslint-disable-line global-require
```
Comment (jleveugle) [inline packages/ovh-manager/webpack.config.js]: Avoid disabling eslint :)

**Review by jleveugle — CHANGES_REQUESTED** [review]

File: packages/ovh-manager/server/proxy/Aapi/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    url: 'http://localhost:8080',
+  },
+};
+
+module.exports = {
+  context: ['/engine/2api'],
+  target: config.aapi.url,
+  changeOrigin: true,
+  pathRewrite: {
+    '^/2api/': '/',
```
Comment (jleveugle) [inline packages/ovh-manager/server/proxy/Aapi/index.js]: to remove i think

**Review by jleveugle — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

**Comment by cbourgois** [comment]
I will change `Aapi` to `aapi`

**Pull request merged**